src/components/todo/TodoItem.tsx

```diff
@@ -62,12 +62,12 @@ const TodoItem = ({ todoItem }: Props) => {
             <Input
               type="text"
               placeholder=""
-              value={todo.todo}
+              value={todo.title}
               onChange={(e) => editFormDataChanger("todo", e.target.value)}
             />
             <Input
               type="date"
-              value={moment(todo.date).format("YYYY-MM-DD")}
+              value={moment(todo.createdAt).format("YYYY-MM-DD")}
```

Comment:
createdAt won't be editable.

Reply:
Kindly add a new field to store task performing date, thanks

Reply:
Please create a separate ticket for that, so that won't block your progress.